geopandas/plotting.py

```diff
@@ -102,9 +106,8 @@ def plot_series(s, colormap='Set1', axes=None, **color_kwds):
     """
     import matplotlib.pyplot as plt
     if axes is None:
-        fig = plt.gcf()
-        fig.add_subplot(111, aspect='equal')
-        ax = plt.gca()
+        fig, ax = plt.subplots(figsize=figsize)
```

Comment:
yes, I also like the new behaviour more. In pandas, DataFrame.plot does also create a new figure (although Series.plot not)